katsdpsigproc/rfi/device.py

```diff
@@ -692,12 +772,16 @@ def generate(**tuning):
                              wgs=[32, 64, 128, 256, 512],
                              vt=[1, 2, 3, 4, 8, 16])
 
-    def instantiate(self, *args, **kwargs):
+    def instantiate(self, command_queue: AbstractCommandQueue,
+                    channels: int, baselines: int, n_sigma: float,
+                    threshold_falloff: float = 1.2,
```

Comment:
It seems like the default value is repeated between here and ThresholdSumDevice.

Reply:
Good point - that's not great. I've made a constant for it.